typescript-data-model/order-history.ts

```diff
@@ -0,0 +1,35 @@
+interface amazonOrder {
+  title: string;
+  date: string;
+  price: number;
+  title2?: string;
+  date2?: string;
+  price2?: number;
```

Comment:
I like that you thought about the one order with two items! The only problem is that this will only handle two items, what if the order has 3, 4, or 5 items? How could we create a list of items the user ordered?

There are also a few more properties we could add such as author, type of item, return date, and the url for the image!